### PR TITLE
Add widget (Issue #1715) - killed/seriously injured in bikes and scooters by year

### PR DESCRIPTION
**Pull request by aentin16**
Widget that returns the number seriously injured or killed in accidents with bicycles, electric bicycles, or scooters per year in a given city.
It is an urban widget as it is only relevant when the newsflash takes place in a city (and not between them) and is included when there have been relevant accidents in that city in the last year. (Ideally in the future - the api of the newsflash would include the vehicles involved, and then it could be included when the newsflash contains one of these vehicles instead).
It returns the seriously injured and fatalities per year per vehicle, plus all three together per year.
Response to issue 1715, Seriously injured & killed in (Bicycles\Electric Bicycles\Electric Scooter) accidents

### CODE REVIEW

**Comment by atalyaalon** [comment]
@aentin16 please fix pylint tests - ask in anyway_backend if you need help with this.

File: anyway/infographics_utils.py [hunk 1]
```diff
@@ -252,6 +252,89 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
...
+                },
+            group_by="accident_year",
+            count="accident_year",
+            start_time=self.request_params.start_time,
+            end_time=self.request_params.end_time,
+        ),
+        "bikes": get_accidents_stats(
+            table_obj=InvolvedMarkerView,
+            filters = {
+                "injury_severity": [1, 2],
```
Comment (ziv17) [inline anyway/infographics_utils.py:283]: Please use constants from backend_constants for values of types like injury_severity and the like.

File: anyway/infographics_utils.py [hunk 2]
```diff
@@ -252,6 +252,89 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
...
 
+@register
+class SevereFatalCountByAccidentYearAndVehicleWidget(UrbanWidget):
+    name: str = "severe_fatal_count_by_vehicle_by_accident_year"
+
+    def __init__(self, request_params: RequestParams):
+        super().__init__(request_params, type(self).name)
+        self.rank = 15
+        self.text = {
+            "title": "Severe or fatal accidents on bikes, e-bikes, or scooters in "
```
Comment (ziv17) [inline anyway/infographics_utils.py:263]: Title, and some other strings needs to be localized. You can look at other widgets code for how to localize.

**Comment by atalyaalon** [comment]
@aentin16 can we close this one?
Did we add all widgets in #1993 ?

**Comment by aentin16** [comment]
@atalyaalon yes, 1993 included all of them. Closed :)

**Pull request closed without merging**